src/ripple/protocol/impl/Rate2.cpp

```diff
@@ -69,6 +69,27 @@ multiplyRound (
 }
 
 STAmount
+multiplyRound (
+    STAmount const& amount,
+    Rate const& rate,
+    Issue const& issue,
+    bool roundUp)
+{
+    assert (rate.value != 0);
+
+    if (rate == parityRate)
+    {
```

Comment:
Just a note that comparable code on line 136 doesn't use the braces.  But it really makes no difference to me.